tokio/src/sync/watch.rs

```diff
@@ -669,6 +669,26 @@ impl<T> Receiver<T> {
         self.version.decrement();
     }
 
+    /// Marks the state as unchanged.
+    ///
+    /// The current value will be considered seen by the receiver.
+    ///
+    /// After calling this method in a single-threaded context, a subsequent call to
+    /// [`has_changed()`](Self::has_changed) will return `false`, and a subsequent call
+    /// to [`changed()`](Self::changed) will yield.
+    ///
+    /// After calling this method in a multi-threaded context, subsequent
+    /// behaviour will vary depending on if a value is sent concurrently.
+    /// For example, a new value could be sent before this method returns
+    /// which would cause a call to [`changed`](Self::changed) to return immediately
```

Comment:
Honestly, I think this is more confusing than it is useful. It's not clear what "single-threaded context" and "multi-threaded context" means. Does code running on a multi-threaded runtime necessarily imply that you are in a "multi-threaded context" according to this paragraph? No, but I anticipate being asked that question in the future. I would rather just take it out entirely.

Reply:
Yeah I'm cool with removing it